test/example_app/app/javascript/packs/hello_react.js

```diff
@@ -1,4 +1,4 @@
-import Hello from 'components/hello';
+import HelloReact from 'components/hello';
 import WebpackerReact from 'webpacker-react';
 
-WebpackerReact.register(Hello);
+WebpackerReact.register({HelloReact});
```

Comment:
I changed `Hello` to `HelloReact` to point out that the class name inside the component does not matter.